### ISSUE TITLE
On image upload/remove there is no indication to screen readers that anything is happening

### ISSUE THREAD

**Issue by srancour** [department-of-veterans-affairs/va.gov-cms#19820]
## User Story or Problem Statement
As a screen reader user I want to know that my file was successfully uploaded or removed.

## Description or Additional Context 
Currently there is only a visual indication that the file was being processed and then uploaded as well as when the file has been removed. There should be some type of indication to a screen reader user that something is happening like in an aria-live region that tells the user about the progress and then when completed the user needs to know that it's done.

## Acceptance Criteria
- [ ] Screen reader hears when an image is being uploaded, removed, and when both are complete
- [ ] Requires accessibility review
